### ISSUE TITLE
ONNX files for Opset 9 cgan and mxnet_arcface get errors for batch_norm_inference, 4d

### ISSUE THREAD

**Issue by mvermeulen** [ROCmSoftwarePlatform/AMDMIGraphX#442]
Running these ONNX files with the MIGraphX driver results in the following errors
```
terminate called after throwing an instance of 'migraphx::version_1::exception'
  what():  /home/mev/source/AMDMIGraphX/src/include/migraphx/check_shapes.hpp:69
: batch_norm_inference: Only 4d supported
```

**Comment by kahmed10** [comment]
cgan has a batchnorm op with input shape (1, 256), which we currently do not support.

**Comment by scxiao** [comment]
both models in this issue passed. Issues related batchnorm are gone
